troposphere/static/js/components/projects/common/SecondaryProjectNavigation.react.js

```diff
@@ -20,9 +20,11 @@ define(function (require) {
 
       var project = this.props.project,
         projectInstances = stores.ProjectInstanceStore.getInstancesFor(project),
+        projectImages = stores.ProjectImageStore.getImagesFor(project),
+        projectExternalLinks = stores.ProjectExternalLinkStore.getExternalLinksFor(project),
         projectVolumes = stores.ProjectVolumeStore.getVolumesFor(project);
 
-      if (projectInstances.length > 0 || projectVolumes.length > 0) {
+      if (projectInstances.length > 0 || projectImages.length > 0  || projectExternalLinks.length > 0 || projectVolumes.length > 0) {
```

Comment:
... though, it maybe be arguably as long as this ... yet it would offer a chance to name what we're checking